src/useLocalStorageState/index_cn.mdx

```diff
@@ -30,13 +34,17 @@ import Demo2CodeJsx from '!raw-loader!./demo/demo2.jsx';
   <Demo2 />
 </JackBox>
 
+<JackBox jsCode={Demo3CodeJsx} tsCode={Demo3CodeTsx} demoName='使用function updater存储' description='useLocalStorageState 里也可以用function updater，就像useState那样.'>
```

Comment:
Space is required between Chinese and English

Reply:
done